firmware/quintuna/CRIT/src/app/app_stackMonitorConfig.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+#include "app_stackMonitor.h"
+
+#include "io_stackMonitorConfig.h"
+
+namespace app
+{
+inline StackMonitor<5> crit_stack_monitor{
```

Comment:
Nit - I think it's cleaner to specify the `stack_monitor_config` type explicitly for all items.